src/xbot/common/command/BaseSubsystem.java

```diff
@@ -13,5 +14,10 @@ public void setDefaultCommand(Command command) {
     protected void initDefaultCommand() {
 
     }
+    
+    public void updateTelemetry() {
```

Comment:
This architecture feels odd. Not all subsystems have telemetry to report and some things that _aren't_ subsystems may want to supply telemetry, so putting it on the subsystem doesn't make much sense to me. Could we define a `TelemetrySource`, `ScheduledTelemetrySource` or similar interface that has this method on it and then implement that on the subsystems that want to be given telemetry cycles? Then the `BaseRobot` would operate on generic `TelemetrySource`s (which could be, for example, my vision server thread).